website/app/components/doc/page/footer.hbs

```diff
@@ -1,6 +1,6 @@
 <footer class="doc-page-footer" ...attributes>
-  <nav class="doc-page-footer__nav-menu" aria-label="footer navigation">
-    <ul class="doc-page-footer__nav-list" role="list">
+  <nav class="doc-page-footer__nav-menu" aria-label="footer">
+    <ul class="doc-page-footer__nav-list" aria-label="footer navigation">
```

Comment:
I tested the page with Safari and VoiceOver, and it seems as though a few things needed tweaking: 
1. the bug that doesn't identify a list as a list in Safari no longer seems to exist
2. added aria-label to the list because otherwise it just is called "list" in the rotor and there's no way to differentiate. 
3. removed the redundant navigation from the nav aria-label because it was reading "footer navigation navigation" (oops!)

Sharing context in case it's useful for reviewers.